1.preprocessing-data/README.md

```diff
@@ -0,0 +1,42 @@
+# 1. Preprocessing CFReT Data
+
+In this module, we present our pipeline for preprocessing the CFReT pilot data.
+
+## Illumination Correction
+
+To correct for illumination issues within the CFReT pilot data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
+We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
+
+Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
+
+Being able to visualize illumination errors can be hard for some datasets.
+As shown in Figure 1 below, the images are brightened using [Fiji](https://imagej.net/software/fiji/) to be able to see the variation in illumination across the image (e.g vignetting).
```

Comment:
```suggestion
To quickly test if illumination correction is necessary for this dataset, we brightened images using [Fiji](https://imagej.net/software/fiji/).
Indeed, we observed technical variation in illumination across the image (e.g vignetting), which is common for Cell Painting data (Figure 1).
```

Just minor suggestions and rewording

Reply:
(also updating to active instead of passive voice)

Reply:
:+1: Time for more active voice practice